gradle.properties

```diff
@@ -2,15 +2,18 @@
 # because some versions here need to be matched by app authors in
 # their own projects. So don't get fancy with syntax!
 
+org.gradle.java.installations.auto-download=false
+org.gradle.jvmargs=-Dfile.encoding=UTF-8
+
 # Versioning
 cordaProductVersion = 5.0.0-INTEROP
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-## IMPORTANT: 
+## IMPORTANT:
 ## The interop feature branches track api revisions separately to the mainline branch.
 ## API version of last merge from corda mainline: 731
```

Comment:
This comment is here so that when merging mainline, we can tell at a glance whether there have been breaking changes in since our last merge. Please update it so that it matches the mainline version that is being merged in for this PR!

Reply:
This doesn't appear to be resolved. Did you push your changes? From the commit history, this should be 742 now.

Reply:
I have pushed it now, but I feel like another manual overhead here. Synching is ongoing process right now with interop branch, so I think we really don't need to maintain it. While merging I don't need to check/validate what was the last version of mainline. Also main branch is also keep updating.  is there any specific requirement for this?

Reply:
It's just a convenience thing. Right now, it's unlikely we'll do a merge from mainline where there isn't a breaking change. But later this may not be the case, and in those cases our api version won't need to be incremented. In that case, I wanted to be able to see what the last merged version was without having to go check the commit history. This way, it's right there above the merge markers and can be done during the merge. It's not a big deal or anything and I'm perfectly happy for it to be removed, but if the comment is present it at very least shouldn't be incorrect.

Also yes, this is a very manual process. Blame the platform team :upside_down_face: